src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/Pileup.java

```diff
@@ -26,13 +26,13 @@
  *
  * <p>This tool emulates the 'samtools pileup' command. It prints the alignment in a format that is very similar to the
  * Samtools pileup format (see the <a href="http://samtools.sourceforge.net/pileup.shtml">Pileup format
- * documentation</a> for more details about the original format). There is one line per genomic position, listing the
+ * documentation</a> for more details about the original format). There is one column per genomic position, listing the
```

Comment:
I think that each line is a genomic position, and then each column shows the information for the list in the following line. I do not really know how to phrase it better, but the current one looks like each column is a genomic position and the first row is the chromosome name, the second the coordinate and so on (transposed format).